_posts/2018-06-26-viroreact.md

```diff
@@ -0,0 +1,643 @@
+---
+layout: post
+authors: [ryan_de_gruyter, michael_vandendriessche]
+title: "ViroReact: Build cross platform AR / VR application for Android and iOS using React"
+image: /img/2018-06-26-viroreact/viroreact.jpg
+tags: [React, React Native, ViroReact, Virtual reality, VR, Augmented reality, AR, Mixed reality, MR]
+category: IoT
+comments: true
+---
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/css/lightbox.css" />
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/bootstrap.css" />
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.9.0/js/lightbox.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap-grid-only@1.0.0/index.min.js"></script>
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Basics of ViroReact](#basics-of-viroreact)
+3. [Demo app](#demo-application)
+4. [Lessons learned and conclusion](#lessons-learned-and-conclusion)
+5. [Extra resources](#extra-resources)
+
+## Introduction
+Augmented Reality is a very interesting space so naturally we wanted to do something with it.
+There are many frameworks available for building cross platform AR/VR applications.
+We came across ViroReact which uses React and React Native to create immersive VR and AR applications using a single codebase for Android and iOS.
+This persuaded us to give it a try.
+In one day, with some tweaking afterwards, we were able to create a simple app to show the status of a meeting room. 
+
+## Basics of ViroReact
+
+### Components
+The main building blocks of a ViroReact app are the components.
+
+#### The most important component is a Scene
+The [scene](https://docs.viromedia.com/docs/scenes) is the digital environment the user interacts with.
+All other components live in this scene.
+```xml
+<ViroScene>
+<ViroARScene>
+```
+
+Just like many other components, the Scene comes in 2 variations: ViroScene and ViroARScene.
```

Comment:
Scene (capital 'S') for consistency.
Two variations
Would wrap ViroScene and ViroARScene in backticks (as they are classes, right?).